recipe/meta.yaml

```diff
@@ -87,7 +81,6 @@ outputs:
         - mkl {{ mkl }}         # [x86]
         - libblas * *_mkl       # [x86]
         - libblas               # [not x86]
-        - libblas               # [not x86]
```

Comment:
As I mentioned in #120, I'm pretty sure that should be `libcblas` (in line with the rundeps)

Reply:
Could you please elaborate as to why?

Reply:
In either case. A PR for this would be appreciated and can be readily merged due to the fact that CPU only builds can be done by the CIs

Reply:
> Could you please elaborate as to why?

Correctness, completeness and symmetry with the run-reqs.

`libcblas` is the C-style interface to BLAS which is (almost certainly) what's actually needed... For MKL specifically this doesn't show up because `libblas` / `libcblas` / `liblapack` / `liblapacke` are all just [symlinks](https://github.com/conda-forge/blas-feedstock/blob/main/recipe/build_pkg.sh#L10) to [`libmkl_rt.so`](https://github.com/conda-forge/blas-feedstock/blob/main/recipe/conda_build_config.yaml#L12), and so the C interface will already be there if just `libblas` (Fortran interface) is specified, but since this is for `# [not x86]`, we should IMO add it.

Reply:
👍🏾